design/ceph/mon-health.md

```diff
@@ -50,11 +50,13 @@ and bring up a new monitor with a new identity. This is an operation that must b
 
 The operator checks for mon health every 45 seconds. If a monitor is down, the operator will wait 5 minutes before failing over the bad mon.
 These two intervals can be configured as parameters to the rook [operator pod](/cluster/examples/kubernetes/ceph/operator.yaml). If the intervals are too short, it could be unhealthy if the mons are failed over too aggressively. If the intervals are too long, the cluster could be at risk of losing quorum if a new monitor is not brought up before another mon fails.
```

Comment:
Still updating this mention of operator.yaml?

Reply:
What do you mean?

Reply:
This doc still mentions that the settings are in the operator.yaml instead of moved to cluster.yaml